pom.xml

```diff
@@ -1095,6 +1095,7 @@
 
             <!-- Used in internal utilities (protected by conditional) -->
             <ignore>java.lang.Long</ignore>
+            <ignore>java.lang.Character</ignore>
```

Comment:
I can do this ... but would be nice to *potentially* get an intrinsic version from the JVM.